src/applications/disability-benefits/2346/containers/App.jsx

```diff
@@ -13,6 +15,30 @@ class App extends Component {
   }
 
   render() {
+    if (
+      // Prevent RUM from running on local/CI environments.
+      !environment.isLocalhost() &&
```

Comment:
```suggestion
      environment.BASE_URL.indexOf('localhost') < 0 &&
```

unfortunately `isLocalhost()` does not achieve the desired result because cypress CI runs on a prod build.